From 5334080cf121a82261f1aa53dcc224f88429530a Mon Sep 17 00:00:00 2001
From: Willy Tarreau <w@1wt.eu>
Date: Mon, 25 Jan 2016 15:27:17 +0100
Subject: [PATCH 30/67] BUG/CLEANUP: CLI: report the proper field states in
 "show sess"

Commit 16f649c ("REORG: polling: rename "fd_spec" to "fd_cache"")
missed the server-facing connection during the rename, so the old
names are still in used and add a bit of confusion during the
debugging.

This should be backported to 1.6 and 1.5.
(cherry picked from commit ceeafb5efc6e916a99aaa59d20e6825b9febc378)
---
 src/dumpstats.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/dumpstats.c b/src/dumpstats.c
index 533f124..e99a7e8 100644
--- a/src/dumpstats.c
+++ b/src/dumpstats.c
@@ -5553,7 +5553,7 @@ static int stats_dump_full_sess_to_buffer(struct stream_interface *si, struct st
 			              obj_base_ptr(conn->target));
 
 			chunk_appendf(&trash,
-			              "      flags=0x%08x fd=%d fd_spec_e=%02x fd_spec_p=%d updt=%d\n",
+			              "      flags=0x%08x fd=%d fd.state=%02x fd.cache=%d updt=%d\n",
 			              conn->flags,
 			              conn->t.sock.fd,
 			              conn->t.sock.fd >= 0 ? fdtab[conn->t.sock.fd].state : 0,
-- 
2.4.10

